src/backend/routes/api.php

```diff
@@ -318,9 +318,10 @@ static function () {
     Route::get('/{serialNumber}', 'EBikeController@show');
     Route::post('/switch', 'EBikeController@switch');
 });
-Route::group(['prefix' => 'export'], static function () {
-    Route::get('/transactions/{slug}', 'TransactionExportController@download');
-    Route::get('/debts/{slug}', 'OutstandingDebtsExportController@download');
+Route::group(['prefix' => 'export', 'middleware' => 'api'], static function () {
+    Route::get('/transactions', 'TransactionExportController@download');
+    Route::get('/debts', 'OutstandingDebtsExportController@download');
```

Comment:
While I like changing the existing endpoints here as well, this does require a frontend change as well. Do adapt to the new routes.

Reply:
Updated the frontend to reflect route changes. Kindly review